compiler/luci/import/src/Nodes/CircleSVDF.cpp

```diff
@@ -0,0 +1,67 @@
+/*
+ * Copyright (c) 2022 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "luci/Import/Nodes/CircleSVDF.h"
+
+#include <luci/IR/Nodes/CircleSVDF.h>
+
+#include <loco.h>
+
+namespace luci
+{
+
+bool CircleSVDFBuilder::validate(const ValidateArgs &args) const
+{
+  const auto &inputs = args.op.inputs;
+  if (!(inputs.size() == 4 || inputs.size() == 5))
+    return false;
+
+  return true;
+}
+
+CircleNode *CircleSVDFBuilder::build_node(const circle::OperatorT &op,
+                                          const std::vector<CircleNode *> &inputs,
+                                          loco::Graph *graph) const
+{
+  auto *node = graph->nodes()->create<CircleSVDF>();
+  node->input(inputs.at(0));
+  node->weight_feature(inputs.at(1));
+  node->weight_time(inputs.at(2));
+  if (inputs.size() == 4)
+  {
+    auto *bias = graph->nodes()->create<CircleOutputExclude>();
+    // CircleOutputExclude doesn't need a type, but since all nodes must have a type,
+    // a dummy type is inserted.
+    bias->dtype(inputs.at(0)->dtype());
+    node->bias(bias);
+
+    node->input_activation_state(inputs.at(3));
+  }
+  else
+  {
+    node->bias(inputs.at(3));
+    node->input_activation_state(inputs.at(4));
+  }
```

Comment:
IIUC there is only 2 possible cases: `inputs.size() == 4` or `inputs.size() == 5`. May be add some assert to input size?

Reply:
I think we could simplify this expression a bit and use `node->input_activation_state(inputs.back());`.

Reply:
My suggestion to this part of code:

```suggestion
  const auto create_output_exclude = [graph]()
  {
    auto *bias = graph->nodes()->create<CircleOutputExclude>();
    // CircleOutputExclude doesn't need a type, but since all nodes must have a type,
    // a dummy type is inserted.
    bias->dtype(loco::DataType::FLOAT32);
    return bias;
  }

  assert(input.size() == 4 || input.size() == 5);
  const auto *bias = (inputs.size() == 4) ? create_output_exclude() : inputs.at(3); 
  node->bias(bias);
  node->input_activation_state(inputs.back());
```

Reply:
Please find my comments above as **optional**. I'm agree with current changes.

Reply:
(1) I prefer not using lambda

(2) `validate` method exist cause not to do `assert(input.size() == 4 || input.size() == 5);` in this method.

Reply:
So, let's leave changes as it is.